File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/17/17.go
// Q17: Print repeated alphabet per row (A, BB, CCC, DDDD).
// Input: An integer n
// Output: Repeated alphabet pattern

package main

import (
    "fmt"
    "strings"
)

func main() {
    var n int
    fmt.Scan(&n)
    for i := 0; i < n; i++ {
        fmt.Println(strings.Repeat(string(rune('A'+i)), i+1))
    }
}

File: Logic Building/2_LOOPING & PATTERNS/4_Pattern Printing/17/17.js
// Q17: Print repeated alphabet per row (A, BB, CCC, DDDD).
// Input: An integer n
// Output: Repeated alphabet pattern

const readline = require('readline');
const rl = readline.createInterface({ input: process.stdin });
rl.on('line', (line) => {
    const n = parseInt(line);
    for (let i = 0; i < n; i++) {
        console.log(String.fromCharCode(65 + i).repeat(i + 1));
    }
    rl.close();
});
